### DISCUSSION TITLE
W Ideas

### DISCUSSION THREAD

**Discussion by SpryzenR** [ScratchAddons/ScratchAddons#6110] [Ideas] [answered]
### Some of my Scratch Addon Ideas:

-show/hide all [sprites, variables, lists] block (Self-Explaintory)
-remove extension option 
-**(!)** change/set all [variables, lists] block 
-**(!)** automatically convert bitmap images to vector option (in costume)
-convert all bitmap images to vector images button that askes if you want to do thi(in costume; I proposed this last time, but I guess I'll just specify)
-remove extension option 

**! = includes a warning to all scratchers who use it on the ScratchAddons website or as a notification confirming they want to do this** 

I don't know if any of this is possible so please tell me because I've been asking myself a lot lately **_"I wish Scratch had [this block] and [this block]"_**

**Scratch Acc for a quicker reply (please follow I love coding and it would make my day):** 
https://scratch.mit.edu/users/SpryzenRYT/

Thank you for your time <3 

-Spryzen

**Comment by CST1229** [comment, marked as answer]
> -show/hide all [sprites, variables, lists] block (Self-Explaintory)
>-(!) change/set all [variables, lists] block

Wouldn't work with users not using the addon, so rejected.

>  -remove extension option

If you don't have any uses of the extension (and didn't show a variable monitor from the extension before), you can reload the project and it will be auto-removed. (Also, you listed this twice.)

**Reply by SpryzenR** [reply]
Ok thanks for letting me know

**Reply by SpryzenR** [reply]
@CST1229 What if the show/hide all [sprites, variables, lists] thing was an option instead of a block?
